packages/runtime/src/widgets/Form/TextInput.tsx

```diff
@@ -149,12 +162,17 @@ export const TextInput: React.FC<TextInputProps> = (props) => {
     return rulesArray;
   }, [values?.validator, values?.mask]);
 
+  const handleChange = (newValue: string) => {
+    setValue(newValue);
+    actionCallback(newValue);
+  };
```

Comment:
Can you combine this callback with the one on line 46

Reply:
Did not look into it before but will look into it right now!